test/specific_parameter_tests.pl

```diff
@@ -22,8 +22,8 @@
 my $ws_name = undef;
 my $ws_client = new Workspace::WorkspaceClient($ws_url,token => $token);
 my $call_back_url = $ENV{ SDK_CALLBACK_URL };
-my $au = new AssemblyUtil::AssemblyUtilClient($call_back_url);
-my $gfu = new GenomeFileUtil::GenomeFileUtilClient($call_back_url);
+my $au = new installed_clients::AssemblyUtilClient($call_back_url);
+my $gfu = new installed_clients::GenomeFileUtilClient($call_back_url);
```

Comment:
Not your error, but these should all be `my $whatever = installed_clients::WhatEverClient->new( $args );` - the correct way to instantiate a class in perl.